dandi/cli/cmd_organize.py

```diff
@@ -34,6 +34,22 @@
     default="auto",
     show_default=True,
 )
+@click.option(
+    "--modify-external-file-fields",
```

Comment:
Suggestion: Rename to `--update-external-file-paths`.

Reply:
https://github.com/dandi/dandi-cli/pull/841/commits/4241edca5a9a434ab9d75a071b60c89af8f34bb5